server/controllers/finance/vouchers.js

```diff
@@ -370,6 +371,10 @@ function safelyDeleteVoucher(guid) {
       WHERE voucher.uuid = ?;
   `;
 
+  const TOGGLE_VOUCHER_REVERSAL = `
+      UPDATE voucher SET voucher.reversed = 0 WHERE voucher.uuid = ?;
+  `;
```

Comment:
Is is correct to have `SET voucher.reversed = 0` for the `TOGGLE_VOUCHER_REVERSAL` instead of `SET voucher.reversed = 1`

Reply:
Yeah, I can see how this is confusing.

The code here is trying to flip the `reversed` flag if the reversing record (i.e. the voucher that made the reverse) is deleted.  So, if you delete the credit note, it will automatically remove the `reversed=1` from the invoice.   The same thing works here for vouchers.